infra/modules/azure_core_infra/variables.tf

```diff
@@ -42,16 +42,12 @@ variable "pep_subnet_cidr" {
 
 variable "vpn" {
   type = object({
-    cidr_subnet              = optional(string, "")
-    dnsforwarder_cidr_subnet = optional(string, "")
+    cidr_subnet              = string
+    dnsforwarder_cidr_subnet = string
+    display_name             = string
   })
-  description = "VPN configuration. Both 'cidr_subnet' and 'dnsforwarder_cidr_subnet' must be specified together or not at all."
+  description = "VPN configuration"
```

Comment:
question: why this change?

Reply:
My understanding is that VPN setup requires CIDR and DNS details and shouldn’t have blank values, but I’d defer to the network specialists for confirmation.